dbt-cta/dbt_project.yml

```diff
@@ -39,7 +39,12 @@ models:
         - cta
       +materialized: incremental
       +on_schema_change: sync_all_columns
-    2_partner_matviews:
+    2_cta_delivery: # for delivery via Analytics Hub subscriptions
+      +tags: 
+        - cta_delivery
+      +materialized: table
+      +full_refresh: true
```

Comment:
oh nice! does this mean we can actually use full_refresh cause we're not making matviews off these tables?

Reply:
yeah! full refresh all the way baby! pretty sure that doesn't break anything in the linked datasets - probably just a very brief blip of unavailability while each table is rebuilding every morning.